tests/containers/install_updates.pm

```diff
@@ -13,14 +13,37 @@ use utils;
 use power_action_utils;
 use version_utils qw(check_os_release get_os_release is_sle);
 
+sub sanity_chk_install {
+    # checks and app-s integration for ongoing work in:
+    #   https://progress.opensuse.org/issues/175950
+    record_info("Sanity check");
+    # cleanup installation repo "dir" type
+    script_run("zypper lr -u | grep 'dir:/run/initramfs' && sudo zypper rr --medium-type dir");
+    # app => package
+    my %pkg = (
+        setfacl => 'acl',
+        firewalld => 'firewalld',
+        tar => 'tar',
+        helm => 'helm',
+    );
+    # check applications for container test
+    foreach my $app (keys %pkg) {
+        my $out = script_run("sudo which $app 2>/dev/null || sudo zypper -n -q in $pkg{$app}");
+        record_info("WARNING", "$app install error", result => 'fail') unless ($out == 0);
+    }
+    return;
+}
+
 sub run {
     my ($self) = @_;
     my $update_timeout = 2400;
     my ($version, $sp, $host_distri) = get_os_release;
 
     # Update the system to get the latest released state of the hosts.
     # Check routing table is well configured
+    record_info("Release", "$host_distri, $version, $sp");
     if ($host_distri =~ /sles|opensuse/) {
+        sanity_chk_install if ($version eq '16');
```

Comment:
please, use `is_sle` function call over here.

Reply:
cannot ue it here, being for BCI containers tests and VERSION is related to containers, not image,

Reply:
So `create_hdd_agama_containers` will run also with different openQA variables than DISTRI=sle and VERSION=16.0?

Reply:
This is the BCI standard, used until 15-SP7 in https://openqa.suse.de/tests/16908392#step/install_updates/2: 
Version number is extracted by [get_os_release](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/95e367e5154e33bb1b1b2ffbe1321c9ce21635bd/tests/containers/install_updates.pm#L19), because HOST_VERSION will refer to host version against VERSION referring to container.